content/tutorial-paychan.md

```diff
@@ -0,0 +1,611 @@
+# Payment Channels Tutorial
+
+Payment Channels are an advanced feature for sending "asynchronous" XRP payments that can be divided into very small increments and settled later. This tutorial walks through the entire process of using a payment channel, with examples using the [JSON-RPC API](reference-rippled.html) of a local `rippled` server.
+
+## Background
+
+The process of using a payment channel always involves two parties, a payer and a payee. The payer is an individual person or institution using the Ripple Consensus Ledger who is a customer of the payee. The payee is a person or business who is doing business on the Ripple Consensus Ledger (RCL), receiving XRP as payment for goods or services.
+
+The types of goods and services are not defined by the software or in this tutorial. However, the types of goods and services that are a good fit for payment channels are:
+
+- Things that can be transmitted near-instantly, like digital items
+- Inexpensive things, where the cost of processing a transaction is a non-trivial portion of the price
```

Comment:
I dunno. Maybe a maker space could charge half a cent for every washer and nail you use from a communal supply?